README_muparser.md

```diff
@@ -0,0 +1,3 @@
+## Math expression evaluation
```

Comment:
I don't think you need to bother with this file - perhaps put the same details into the commit messages?